terraform/a2ha-terraform/reference_architectures/aws/main.tf

```diff
@@ -62,7 +59,7 @@ module "aws" {
   lb_access_logs                     = var.lb_access_logs
   tags                               = var.aws_tags
   aws_instance_profile_name          = var.backup_config_s3 == "true" ? module.s3[0].instance_profile_name : null
-  json_data                          = local.json_data
+  json_data                          = module.aws_metadata.json_data
```

Comment:
Inconsistent value assignment for the key json_data between the main.tf files in destroy folder and ref. architecture folder. Is that intentional?

Reply:
Modified!
 Previously we have different json_data for destroy and ref architecture. So I have added the same.